src/ert/gui/simulation/simulation_panel.py

```diff
@@ -144,10 +144,11 @@ def runSimulation(self):
             abort = False
             QApplication.setOverrideCursor(Qt.CursorShape.WaitCursor)
             try:
+                config = self.ert.ert_config
                 args = self.getSimulationArguments()
                 experiment = self._notifier.storage.create_experiment(
-                    parameters=self.ert.ensembleConfig().parameter_configuration,
-                    responses=self.ert.ensembleConfig().response_configuration,
+                    parameters=config.ensemble_config.parameter_configs,
+                    responses=config.ensemble_config.response_configuration,
```

Comment:
Can we limit the scope of the try block here?

Reply:
Probably, a bit outside the scope I was originally going for, but can change it, where do you want it?

Reply:
To only cover the part of the code that we think will throw a `KeyError`